lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1740,18 +1741,10 @@ namespace Js
         }
 
         //8. Let descObj be FromPropertyDescriptor(Desc).
-        //9. NOTE If Desc was originally generated from an object using ToPropertyDescriptor, then descObj will be that original object.
-        //10. Let trapResult be the result of calling the[[Call]] internal method of trap with handler as the this value and a new List containing target, P, and descObj.
-        //11. Let booleanTrapResult be ToBoolean(trapResult).
-        //12. ReturnIfAbrupt(booleanTrapResult).
-        //13. If booleanTrapResult is false, then return false.
-        //14. Let targetDesc be the result of calling the[[GetOwnProperty]] internal method of target with argument P.
-        //15. ReturnIfAbrupt(targetDesc).
-        Var descVar = descriptor.GetOriginal();
-        if (descVar == nullptr)
```

Comment:
The removal of this seems a bit strange (or maybe the fact that it was present seems a bit strange -- not sure which) -- anyway, it looks like it was based on the NOTE which was removed from the spec, and it reads in such a way that it was redundant anyway?

Reply:
I think it's presence in the first place was an error - see the test case I added or case within #5680 for the problem this could cause.